src/engine/websites/MangaBuddy.ts

```diff
@@ -0,0 +1,24 @@
+import { Tags } from '../Tags';
+import icon from './MangaBuddy.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+
+@Common.MangaCSS(/^https?:\/\/mangabuddy\.com\/[^/]+$/, 'div.name.box h1')
+@Common.MangasMultiPageCSS('/az-list?page={page}', 'div.manga-list div.title h3 a', 1)
+@Common.ChaptersSinglePageCSS('ul.chapter-list li a', Common.AnchorInfoExtractor(false, '.chapter-update'))
+@Common.PagesSinglePageJS(`      new Promise(resolve => {

```

Comment:
In case the script may become too large, consider extracting it into a constant variable placed before the decorators
e.g.,
```javascript
const scriptImageLinks = `
    new Promise(resolve => {
        const images = window.chapImages.split(',');
        resolve(images.map(image => window.mainServer + image));
    });
`;

@Common.PagesSinglePageJS(scriptImageLinks);
```